newIDE/app/src/ProjectCreation/CreateProjectDialog.js

```diff
@@ -9,8 +9,24 @@ import Tutorials from './Tutorials';
 import { Column } from '../UI/Grid';
 import { type StorageProvider, type FileMetadata } from '../ProjectsStorage';
 
+class StringStore {
```

Comment:
Not sure if that is the best way, if I should have declared that in a separate file and if there wasn't an equivalent already

Reply:
You don't need this complex "string store". Just store a string in the state :)

Reply:
Your custom string store won't notify React that something changed, because you won't call React `setState`, so the component will not re-render when there is a change,